src/utils/LoggerLevels.ts

```diff
@@ -0,0 +1,6 @@
+export enum LoggerLevel {
+  debug = 3,
+  info = 2,
+  warn = 1,
+  error = 0,
+}
```

Comment:
nit: we haven't been very consistent so far, I propose the following which matches what a lot of TypeScript projects are doing.
```suggestion
export enum LoggerLevel {
  Debug = 3,
  Info = 2,
  Warn = 1,
  Error = 0,
}
```